terminal/ansi.go

```diff
@@ -73,11 +89,18 @@ func ansiHandler(pty chan rune, terminal *Terminal) error {
 }
 
 func nextLineHandler(pty chan rune, terminal *Terminal) error {
+	terminal.Lock()
+	defer terminal.Unlock()
+
 	terminal.ActiveBuffer().NewLineEx(true)
 	return nil
 }
 
 func tabSetHandler(pty chan rune, terminal *Terminal) error {
+	// Handler should lock the terminal if there will be write operations to any data read by the renderer
+	// terminal.Lock()
+	// defer terminal.Unlock()
```

Comment:
Does this need to stay here?

Reply:
This is a reminder. It's left here intentionally.